include/cupla/cudaToCupla/driverTypes.hpp

```diff
@@ -68,24 +64,58 @@
 
 // index renaming
 #define blockIdx                                                               \
-  static_cast<uint3>(                                                \
+  static_cast<cupla::uint3>(                                                \
```

Comment:
I'm sorry, something went wrong this and with the above one. I thought I explicitly solved the merge-conflict in favor of the original.